pkg/services/ngalert/api/tooling/spec.json

```diff
@@ -781,6 +781,89 @@
         }
       }
     },
+    "/api/v1/ngalert/configuration": {
+      "get": {
+        "produces": [
+          "application/json"
+        ],
+        "tags": [
+          "configuration"
+        ],
+        "summary": "Get the NGalert configuration of the user's organization, returns 404 if no configuration is present.",
+        "operationId": "RouteGetNGalertConfig",
+        "parameters": [
+          {
+            "name": "Body",
+            "in": "body",
+            "schema": {
+              "$ref": "#/definitions/PostableNGalertConfig"
+            }
+          }
+        ],
+        "responses": {
+          "200": {
+            "$ref": "#/responses/GettableNgalertconfig"
```

Comment:
I get the following errors when I load the spec:
```
Errors
Hide
 
Semantic error at paths./api/v1/ngalert/configuration.get.responses.200.$ref
$refs must reference a valid location in the document
Jump to line 3835
Resolver error at paths./api/v1/ngalert/configuration.get.responses.200.$ref
Could not resolve reference: Could not resolve pointer: /responses/GettableNgalertconfig does not exist in document
Jump to line 3835
```

Also, can you upload only the changes related to the new endpoints because due to the spec generation issues some responses appear empty:
<img width="696" alt="Screenshot 2021-08-05 at 12 29 45 PM" src="https://user-images.githubusercontent.com/1632407/128327597-45bbffa4-dacd-418c-bf5e-ed94e1a40a81.png">
instead of:
<img width="610" alt="Screenshot 2021-08-05 at 12 36 21 PM" src="https://user-images.githubusercontent.com/1632407/128328415-a9cac822-3e01-443e-ab5e-31b1b6511252.png">

Reply:
Let me take a look at this - I'm having trouble running the scripts in Apple Silicon so there's a chance this might be all wrong.

Reply:
Let me know if you prefer me to run it and push the changes.
BTW your are not the only one reporting this; what kind of errors you get?

Reply:
I think I've fixed this now after a few hurdles - can you double check?